layouts/_default/list.html

```diff
@@ -17,7 +17,7 @@ <h2 class="post-title"><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
 		  {{ if .Params.tags }}
 			<span class="post-tags">
 		  	{{ range .Params.tags }}
-			          #<a href="/tags/{{ . | urlize }}">{{ . }}</a>&nbsp;
+			          #<a href="{{ $.Site.BaseURL }}/tags/{{ . | urlize }}">{{ . }}</a>&nbsp;
```

Comment:
need to remove `/` if we use `{{ $.Site.BaseURL }}` because it will introduce `/` at the end of base url.